addons/hr_attendance/controllers/main.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import http
+from odoo.http import request
+
+
+class HrAttendance(http.Controller):
+    @http.route('/hr_attendance/kiosk_keepalive', auth='none', type='json')
```

Comment:
Better to limit access

```suggestion
    @http.route('/hr_attendance/kiosk_keepalive', auth='user', type='json')
```